spec/models/rubric_criterion_spec.rb

```diff
@@ -143,7 +143,9 @@
     context 'when parsing a CSV file' do
       describe 'raise csv line error on rows with elements without names for every criterion' do
         row = %w[name 1.0]
-        (0..RubricCriterion::RUBRIC_LEVELS - 2).each do |i|
+        # get number of required fields
+        required = Level.validators.grep(ActiveRecord::Validations::PresenceValidator).length
```

Comment:
You should change this test (see my comment about the CSV row format above).

Reply:
Because we are only checking to see if the first input raises an error, can I hard code a number into the test?